CHANGELOG.md

```diff
@@ -4,6 +4,29 @@
 
 - "You miss 100 percent of the chances you don't take. — Wayne Gretzky" — Michael Scott
 
+## 7.31.0
+
+- feat(nextjs): Add Edge Runtime SDK (#6752)
+- feat(nextjs): Add optional options argument to `withSentryConfig` as an alternative to the `sentry` property (#6721)
+- feat(nextjs): Add edge route and middleware wrappers (#6771)
+- feat(nextjs): Auto-wrap edge-routes and middleware (#6746)
+- feat(replay): Change `addEvent` to be async (#6695)
```

Comment:
```suggestion
- ref(replay): Change `addEvent` to be async (#6695)
```

This is actually more of an internal refactor, maybe we can even remove this completely.

Reply:
Let's remove entirely.